src/lib/viewers/doc/_docBase.scss

```diff
@@ -252,14 +252,15 @@ $thumbnail-sidebar-width: 226px;
     .textLayer {
         top: 15px; // Match 15px padding top on page
         bottom: auto;
+        opacity: 1;
```

Comment:
Why is this needed?

Reply:
This is needed because pdfjs styles it with `opacity: .2` so all colors get .2 transparency. But the colors we'll be applying need more control than that

Reply:
I suspect this value is set by pdfjs as a default for a reason, though. Does it have any consequences on the overall user experience? It looks like the test document is mostly text with few images.

Reply:
Looking at https://github.com/mozilla/pdfjs-dist/blob/c17c20fb126fcb284811616662156022bbde1947/web/pdf_viewer.css, the only elements of the .textLayer that is affected by this `opacity` was the find bar's highlights which is what I tried to account for below

Reply:
Gotcha. So long as the text layer itself doesn't have a background color or something, it should be okay.